rocketpy/Flight.py

```diff
@@ -2349,6 +2400,30 @@ def allInfo(self):
 
         plt.subplots_adjust(hspace=0.5)
         plt.show()
+        return None
+
+    def angularPositionPlots(self):
```

Comment:
Might be fair to combine this function with the one that plots angular velocities and accelerations